src/org/usfirst/frc/team2585/Environment.java

```diff
@@ -0,0 +1,40 @@
+package org.usfirst.frc.team2585;
+
+import org.impact2585.lib2585.RobotEnvironment;
+import org.usfirst.frc.team2585.input.InputMethod;
+import org.usfirst.frc.team2585.input.XboxInput;
+import org.usfirst.frc.team2585.systems.WheelSystem;
+
+
+public class Environment extends RobotEnvironment{
+	
+	/**
+	 * 
+	 */
+	//TODO: change to an appropriate value
+	private static final long serialVersionUID = 1L;
+	
+	private InputMethod input;
+	private WheelSystem wheels;
+	
+	public Environment(Robot robot) {
+		super(robot);
+		wheels = new WheelSystem();
```

Comment:
Don't forget to call the init function for WheelSystem. You can either do it here or do it in the constructor.